package.json

```diff
@@ -94,6 +95,11 @@
                 "category": "Azure App Service"
             },
             {
+                "command": "appService.DeploymentScript",
+                "title": "Generate Deployment Script",
```

Comment:
Can we change this name and/or the grouping? Right now it sounds like you're creating a script to deploy _to_ the app service (rather than deploying _the_ app service)

Maybe `Generate Creation Script` (Since we call the other command `Create Web App`)